apis/Google.Cloud.Dlp.V2/Google.Cloud.Dlp.V2/DlpGrpc.cs

```diff
@@ -71,8 +71,9 @@ public static partial class DlpService
     static readonly grpc::Marshaller<global::Google.Cloud.Dlp.V2.ListJobTriggersRequest> __Marshaller_ListJobTriggersRequest = grpc::Marshallers.Create((arg) => global::Google.Protobuf.MessageExtensions.ToByteArray(arg), global::Google.Cloud.Dlp.V2.ListJobTriggersRequest.Parser.ParseFrom);
     static readonly grpc::Marshaller<global::Google.Cloud.Dlp.V2.ListJobTriggersResponse> __Marshaller_ListJobTriggersResponse = grpc::Marshallers.Create((arg) => global::Google.Protobuf.MessageExtensions.ToByteArray(arg), global::Google.Cloud.Dlp.V2.ListJobTriggersResponse.Parser.ParseFrom);
     static readonly grpc::Marshaller<global::Google.Cloud.Dlp.V2.DeleteJobTriggerRequest> __Marshaller_DeleteJobTriggerRequest = grpc::Marshallers.Create((arg) => global::Google.Protobuf.MessageExtensions.ToByteArray(arg), global::Google.Cloud.Dlp.V2.DeleteJobTriggerRequest.Parser.ParseFrom);
-    static readonly grpc::Marshaller<global::Google.Cloud.Dlp.V2.CreateDlpJobRequest> __Marshaller_CreateDlpJobRequest = grpc::Marshallers.Create((arg) => global::Google.Protobuf.MessageExtensions.ToByteArray(arg), global::Google.Cloud.Dlp.V2.CreateDlpJobRequest.Parser.ParseFrom);
+    static readonly grpc::Marshaller<global::Google.Cloud.Dlp.V2.ActivateJobTriggerRequest> __Marshaller_ActivateJobTriggerRequest = grpc::Marshallers.Create((arg) => global::Google.Protobuf.MessageExtensions.ToByteArray(arg), global::Google.Cloud.Dlp.V2.ActivateJobTriggerRequest.Parser.ParseFrom);
```

Comment:
There's no corresponding change to the GAPIC client; so I assume this method hasn't been added to the gapic config yaml.
Is this expected?

Reply:
Hadn't spotted that. I don't know whether it's intentional, but I think it's at least safe.